### ISSUE TITLE
Error: bad valence on Ar

### ISSUE THREAD

**Issue by sitanshubhunia** [epam/Indigo#697]
Database: "PostgreSQL 12.7, compiled by Visual C++ build 1914, 64-bit"
Bingo version: "1.9.1.r0-g353401f win64"

Following error is getting from **CheckReaction** method

method is return :
**element: bad valence on Ar having 1 drawn bonds, charge 0, and 0 radical electrons**
![Bad_valence_onAr_pg](https://user-images.githubusercontent.com/840684/160757541-3d9e517a-3eff-4c9c-84ed-9b5b67939e86.png)

Please find the cml file of that reaction 
[Ar_issue.cml.txt](https://github.com/epam/Indigo/files/8377221/Ar_issue.cml.txt)

****Is it possible to fix this issue with change in any parameter(s)  on bingo_config table ?** or any other way to fix it without change in reaction.**

Basically I wants to store this reaction as it is and search it accordingly.

**Comment by even1024** [comment]
Hello. We will check it and try to to find solution for the issue.

**Comment by AlexanderSavelyev** [comment]
HI @sitanshubhunia , have you tried to change the 'IGNORE_BAD_VALENCE' in bingo_config to '1' ?

**Comment by sitanshubhunia** [comment]
Hi @AlexanderSavelyev 
**Thanks for your help. It did work.** 
After changing  'IGNORE_BAD_VALENCE' in bingo_config to '1' , index created.

Now I have face another one
FYI
**molecule: allowRGroupOnRSite(): atom #0 is not an r-site**
Reaction is attached for your kind attention.
 
[allowRGroupOnRSite.cml.txt](https://github.com/epam/Indigo/files/8407502/allowRGroupOnRSite.cml.txt)

**It would be better if any Documentation on  bingo_config table is available with example.**
 
**In the near future, I would be in need for such kind of grateful assistance from your end.**

**Comment by AlexanderSavelyev** [comment]
I have created #701 and #702. Closing the ticket
